### PR TITLE
WINC-1036: Drop Azure support in WMCO 5.x and lower

### PR DESCRIPTION
**Pull request by aravindhp**
Add an explanation of why we are dropping Azure support in WMCO 5.x and lower.

### CODE REVIEW

**Comment by aravindhp** [comment]
retitle: WINC-1036: Drop Azure support in WMCO 5.x and lower

**Comment by aravindhp** [comment]
/retitle WINC-1036: Drop Azure support in WMCO 5.x and lower

File: docs/wmco-prerequisites.md [hunk 1]
```diff
@@ -6,11 +6,17 @@ may be relevant.
 | Cloud Provider            | Supported OKD/OCP Version | Supported WMCO version |
 |---------------------------|---------------------------|------------------------|
 | Amazon Web Services (AWS) | 4.6+                      | WMCO 1.0+              |
-| Azure                     | 4.6+                      | WMCO 1.0+              |
+| Azure                     | 4.11+                     | WMCO 6.0+              |
 | GCP                       | 4.12+                     | WMCO 7.0+              |
 | VMware vSphere            | 4.7+                      | WMCO 2.0+              |
 | Platform none (BYOH)      | 4.8+                      | WMCO 3.1.0+            |
 
+Note: We added Azure support in 4.6 but given Microsoft has [stopped publishing Windows Server 2019 images with
```
Comment (mburke5678) [inline docs/wmco-prerequisites.md:14]:
```suggestion
Note: We added Azure support in 4.6 but given that Microsoft has [stopped publishing Windows Server 2019 images with
```

File: docs/wmco-prerequisites.md [hunk 2]
```diff
@@ -6,11 +6,17 @@ may be relevant.
...
+Docker](https://techcommunity.microsoft.com/t5/containers/important-update-deprecation-of-docker-virtual-machine-images/ba-p/3646272),
+we have decided to drop Azure support for releases older than 6.x. From 6.0.0 onwards we are using containerd as the
```
Comment (mburke5678) [inline docs/wmco-prerequisites.md:16]:
```suggestion
we have decided to drop Azure support for releases older than 6.x. For 5.x and below it was a requirement for the Windows Server 2019 images to have Docker pre-installed.  From 6.0.0 onwards we are using containerd as the
```

File: docs/wmco-prerequisites.md [hunk 3]
```diff
@@ -6,11 +6,17 @@ may be relevant.
...
+runtime and it is WMCO's responsibility to manage that. This is unlike 5.x and below where there was a requirement for
+the Windows Server 2019 images to have Docker pre-installed.
```
Comment (mburke5678) [inline docs/wmco-prerequisites.md:18]: Remove:  This is unlike 5.x and below where there was a requirement for
the Windows Server 2019 images to have Docker pre-installed.

File: docs/wmco-prerequisites.md (hunk 1 above)
Comment (aravindhp) [reply docs/wmco-prerequisites.md]: Done.

File: docs/wmco-prerequisites.md (hunk 2 above)
Comment (aravindhp) [reply docs/wmco-prerequisites.md]: Done.

File: docs/wmco-prerequisites.md (hunk 3 above)
Comment (aravindhp) [reply docs/wmco-prerequisites.md]: Done.

**Review by aravindhp — COMMENTED** [review]
Thanks for the review @mburke5678. I have addressed your comments.

File: docs/wmco-prerequisites.md [hunk 4]
```diff
@@ -6,11 +6,17 @@ may be relevant.
 | Cloud Provider            | Supported OKD/OCP Version | Supported WMCO version |
 |---------------------------|---------------------------|------------------------|
 | Amazon Web Services (AWS) | 4.6+                      | WMCO 1.0+              |
-| Azure                     | 4.6+                      | WMCO 1.0+              |
+| Azure                     | 4.11+                     | WMCO 6.0+              |
 | GCP                       | 4.12+                     | WMCO 7.0+              |
 | VMware vSphere            | 4.7+                      | WMCO 2.0+              |
 | Platform none (BYOH)      | 4.8+                      | WMCO 3.1.0+            |
```
Comment (saifshaikh48) [inline docs/wmco-prerequisites.md:12]: Thoughts on making the table more chronological?
```
| Cloud Provider            | Supported OKD/OCP Version | Supported WMCO version |
|---------------------------|---------------------------|------------------------|
| Amazon Web Services (AWS) | 4.6+                      | WMCO 1.0+              |
| VMware vSphere            | 4.7+                      | WMCO 2.0+              |
| Platform none (BYOH)      | 4.8+                      | WMCO 3.1.0+            |
| Azure                     | 4.11+                     | WMCO 6.0+              |
| GCP                       | 4.12+                     | WMCO 7.0+              |
```

Comment (aravindhp) [reply docs/wmco-prerequisites.md:12]: I could but that would have made reviewing this PR more difficult. But I can do this.

Comment (aravindhp) [reply docs/wmco-prerequisites.md]: Done.

**Pull request merged**